code/modules/flufftext/Hallucination.dm

```diff
@@ -1,420 +1,292 @@
-/*
-Ideas for the subtle effects of hallucination:
-
-Light up oxygen/phoron indicators (done)
-Cause health to look critical/dead, even when standing (done)
-Characters silently watching you
-Brief flashes of fire/space/bombs/c4/dangerous shit (done)
-Items that are rare/traitorous/don't exist appearing in your inventory slots (done)
-Strange audio (should be rare) (done)
-Gunshots/explosions/opening doors/less rare audio (done)
-
-*/
-
-mob/living/carbon/var
-	image/halimage
-	image/halbody
-	obj/halitem
-	hal_screwyhud = 0 //1 - critical, 2 - dead, 3 - oxygen indicator, 4 - toxin indicator
-	handling_hal = 0
-	hal_crit = 0
-
-mob/living/carbon/proc/handle_hallucinations()
+//Health hud screws for carbon mobs
+#define SCREWYHUD_NONE 0
+#define SCREWYHUD_CRIT 1
+#define SCREWYHUD_DEAD 2
+#define SCREWYHUD_HEALTHY 3
+
+GLOBAL_LIST_INIT(hallucination_list, list(
+	/datum/hallucination/sounds = 100,
+	/datum/hallucination/chat = 50,
+	/datum/hallucination/battle = 20,
+	/datum/hallucination/xeno_attack = 12,
+	/datum/hallucination/death = 1,
+	))
+
+/mob/living/carbon
+	// halucination vars
+	var/image/halimage
+	var/image/halbody
+	var/obj/halitem
+	var/hal_screwyhud = SCREWYHUD_NONE
+	var/next_hallucination = 0
+	var/cpr_time = 1 //CPR cooldown.
```

Comment:
Could you place these variable on the carbon definition?

Reply:
Yeah it was